test/tests/functional/pbs_qsub_opts_args.py

```diff
@@ -56,6 +56,22 @@ def setUp(self):
         self.fn = self.du.create_temp_file(body=script)
         self.qsub_cmd = os.path.join(
             self.server.pbs_conf['PBS_EXEC'], 'bin', 'qsub')
+        self.jobdir_root = "/tmp"
+        self.jobdir = []
+        self.remove_jobdir = False
+
+    def tearDown(self):
+        if self.remove_jobdir:
+            for mom in self.moms.values():
+                for d in self.jobdir:
+                    if d.startswith(self.jobdir_root):
+                        self.logger.info('%s:remove jobdir %s' % (
+                                         mom.hostname, d))
+                        cmd = ['/bin/rm', '-rf', d]
+                        self.du.run_cmd(mom.hostname, cmd=cmd,
+                                        sudo=True)
+            self.remove_jobdir = False
+        TestFunctional.tearDown(self)
```

Comment:
Will removing the directories cause any problems when the jobs are removed?  If so, remove them after TestFunctional.tearDown()

Reply:
Yes, TestFunctional.tearDown() can be called first.